docs/source/datasets.rst

```diff
@@ -0,0 +1,196 @@
+.. _datasets:
+
+⭐ Table of supported datasets ⭐
+=================================
+
+This table is provided as a guide for users to select appropriate datasets. The
+list of annotations omits some metadata for brevity, and we document the dataset's
+primary annotations only. The number of tracks indicates the number of unique "tracks"
+in a dataset, but it may not reflect the actual size or diversity of a dataset,
+as tracks can vary greatly in length (from a few seconds to a few minutes),
+and may be homogeneous. For specific information about the contents of each dataset,
```

Comment:
"may be homogeneous" as in: it might be a lot of the same.
for example, beatles is more homogeneous than medleydb. So it's better (more interesting) to evaluate on 100 MDB tracks than on 100 Beatles track.